imports/api/PointCalculationUtils.js

```diff
@@ -0,0 +1,118 @@
+import { Constants } from './Constants';
+import { NewGameUtils } from './NewGameUtils';
+
+export var PointCalculationUtils = {
+    jpn: {
+        "dealin_delta": dealin_delta,
+        "selfdraw_delta": selfdraw_delta,
+        "fuckup_delta": fuckup_delta
+    }
+};
+
+function dealin_delta(points, fu, winnerWind, loserWind, riichiSticks) {
+    let winds = {};
+    winds[Constants.EAST] = winds[Constants.SOUTH] = winds[Constants.WEST] = winds[Constants.NORTH] = 0;
+
+	let basicPoints;
+    let multiplier = (winnerWind != NewGameUtils.roundToDealerDirection(Number(Session.get("current_round")))) ? 4 : 6;
+    let manganPayout = multiplier * Constants.MANGAN;
+
+	if (points < 5) {
+        if (fu == 20 || (points == 1 && fu == 25)) {
```

Comment:
Super funky indentation.  Use tabs for indentation and spaces for alignment please.